cluster_test.go

```
@@ -11410,3 +11410,428 @@ func clusterClientWithConnCount(n int) *clusterClient {
+	mi, si, ei := -1, -1, -1
+	for i, cm := range moved.commands {
+		switch strings.Join(cm.Commands(), " ") {
+		case "MULTI":
+			mi = i
+		case "SET {t}k v":
+			si = i
+		case "EXEC":
+			ei = i
+		}
+	}
+	if !(mi >= 0 && si == mi+1 && ei == si+1) {
```

golangci-lint (2.12.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)